apps/music/js/ui/views/player_view.js

```diff
@@ -775,10 +777,22 @@ var PlayerView = {
   },
 
   setSeekBar: function pv_setSeekBar(startTime, endTime, currentTime) {
+    if (this.seekBar.max != endTime) {
+      // Duration changed, update accessibility label.
+      navigator.mozL10n.setAttributes(this.seekSlider,
+        'playbackSeekBar', {'duration': formatTime(endTime)});
+    }
+
```

Comment:
@eeejay Music app has listened to `durationchange` event of the audio element, so if the duration attribute should be updated when the duration changes, it might make more sense to place it in the `durationchange` handler.